package.json

```diff
@@ -13,7 +13,7 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "broccoli build temp && mocha"
+    "test": "rm -rf temp tmp && broccoli build temp && mocha"
```

Comment:
Use [`trash`](https://github.com/sindresorhus/trash) so it works cross-platform.